qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -183,7 +183,45 @@ def _get_version(self):
             mes = self.ask(bytes([3, 4]))
             ver = mes[2]
             return ver
-
+    
+    def linsweep(self,start,end,points):
+        """
+        A replacement of numpy.linspace for use with the ivvi rack dacs.
+        ivvi.linsweep gives:
+         - even spacing in voltages
+         - voltages match the set voltages
+         - protection from oversampling
+         
+        Args:
+            start (number): starting voltage
+            end (number): ending voltage
+            points (int): number of points between start and end, inclusive
+            
+        Returns:
+            list of valid voltages for the ivvi rack
+            
+        Examples:
+            #normal dac aligned voltages
+            ivvi.linsweep(-200,200,201) -> [ -199.404, -197.391 ... ] 
+            
+            #protection from oversampling
+            ivvi.linsweep(start=108.8,end=109,points=11) -> 
```

Comment:
is this really desired behavior? the caller requests 11 points (`..points=11`) but gets 4? If i was the user, i'd raise an exception in this case. but I am definitely not fully aware how this method is supposed to be used.